src/app/view/components/_common/dot-add-to-bundle/dot-add-to-bundle.component.html

```diff
@@ -0,0 +1,26 @@
+<p-dialog
+    modal="modal"
+    width="500"
+    [dismissableMask]="true"
+    [visible]="true"
+    [responsive]="true"
+    [resizable]="false"
+    [draggable]="false"
+    (onHide)="close($event)"> 
+
+    <p-header>
+        {{dotMessageService.get('contenttypes.content.add_to_bundle')}}
+    </p-header>
+
+    <form [formGroup]="form" (ngSubmit)="submitBundle($event)" novalidate #formEl="ngForm">
+        <div class="form__group">
+            <p-dropdown [style]="{'width':'100%'}" [options]="bundle$ | async" editable="true" formControlName="addBundle" [placeholder]="placeholder" optionLabel="name"></p-dropdown>
```

Comment:
Do we need this 100%?

Reply:
Yes, it needs to have the width: 100% to match the mock.

Reply:
All the form fields in a form will be 100% width (all forms will be vertical) so add this 100% global to this component.